bin/webpack-wizard-dev.js

```diff
@@ -14,9 +14,9 @@ const webpackConfig = webpackWizard(webpackWizardConfig);
 const compiler = webpack(webpackConfig);
 
 const app = express();
-const host = webpackWizardConfig.devHost || 'localhost';
-const port = webpackWizardConfig.devPort || 3000;
-const htmlPath = webpackWizardConfig.input.html || path.resolve(process.cwd(), 'index.html');
+const host = webpackWizardConfig.devHost;
```

Comment:
If `readConfig` returns a full config with defaults, then these three variables no longer serve any purpose.

Reply:
Variables `host` and `port` brings cleaner code in lines below.
Variable `htmlPath` removed.